### ISSUE TITLE
package javax.annotation does not exist

### ISSUE THREAD

**Issue by spilehchiha** [mwcvitkovic/Deep_Learning_On_Code_With_A_Graph_Vocabulary--Code_Preprocessor#4]
While trying to run the command` mvn install -DskipTests`, I receive the error `package javax.annotation does not exist` for a lot of files.
I also tried to include this dependency in the `pom.xml ` file in casein was not there, and it still didn't work:
`<dependency>
                <groupId>javax.annotation</groupId>
                <artifactId>javax.annotation-api</artifactId>
                <version>1.2</version>
            </dependency>`
I was first using Java 12, but even when I use Java 8 (which seems to solve the problem for a few), I still get the same error, plus an additional error for the symbol `additionalOption` in the `pom.xml` file not being resolved.

**Comment by mwcvitkovic** [comment]
@badalgeek

**Comment by spilehchiha** [comment]
As a matter of fact, there is issue with that package.
I had some problems on conflicts between my Java versions.
My default Java version was 12 and I had trouble converting it to JDK 1.8.
I re-installed my system, and downloaded  a brand-new JDK from Oracle:
`java version "1.8.0_221"
Java(TM) SE Runtime Environment (build 1.8.0_221-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.221-b11, mixed mode)`

This is also the version of Maven I installed:
`Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T15:00:29-04:00)`

After that, I followed the instructions and the .gml files were produced successfully.

Although I think it's better to edit the processDatasets.sh file in a way to produce the format of the output files as .graphml and not .gml, since it seems that software like Gephi gets confused and issues errors while trying to open them for inspection purposes. (probably because GML stands for another format with a different structure, namely Graph Modeling Language, also supported by Gephi).
